Update technologies.md

dannoakes opened the pull request:
Adding the Windows technologies used.

dannoakes committed:
Update technologies.md

Adding the Windows technologies used.

ghost commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/microsoft-365-docs?pullRequest=3380) <br/>All CLA requirements met.

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 commented on microsoft-365/managed-desktop/intro/technologies.md:
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
+Recommended
+[Windows Hello for Business](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-identity-verification) | Customers are recommended to implement Windows Hello for Business to replace passwords with strong two-factor authentication used on Microsoft Managed Desktop devices.
+[Application Virtualization](https://docs.microsoft.com/windows/application-management/app-v/appv-technical-reference) | Customers can deploy Application Virtualization (App-V) packages using the Intune Win32 app management client.
+[Microsoft 365 data loss prevention](https://docs.microsoft.com/microsoft-365/compliance/endpoint-dlp-learn-about) | Customers are recommend to implement Microsoft 365 data loss prevention (DLP) to monitor the actions that are being taken on items you've determined to be sensitive and to help prevent the unintentional sharing of those items.   
```
```suggestion
[Microsoft 365 data loss prevention](https://docs.microsoft.com/microsoft-365/compliance/endpoint-dlp-learn-about) | Customers are recommended to implement Microsoft 365 data loss prevention (DLP) to monitor the actions that are being taken on items you've determined to be sensitive and to help prevent the unintentional sharing of those items.   
```

JohanFreelancer9 commented on microsoft-365/managed-desktop/intro/technologies.md:
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
+
+Included in the service plan
+[BitLocker Drive Encryption](https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview) | BitLocker Drive Encryption is used to encrypt all system drives. 
+[Windows Defender System Guard]( https://docs.microsoft.com/windows/security/threat-protection/windows-defender-system-guard/system-guard-how-hardware-based-root-of-trust-helps-protect-windows) | Protects the integrity of the system at start up and validates that system integrity has truly been maintained.
```
```suggestion
[Windows Defender System Guard]( https://docs.microsoft.com/windows/security/threat-protection/windows-defender-system-guard/system-guard-how-hardware-based-root-of-trust-helps-protect-windows) | Protects the integrity of the system at start-up and validates that system integrity has truly been maintained.
```

JohanFreelancer9 commented on microsoft-365/managed-desktop/intro/technologies.md:
```suggestion
[Microsoft Defender Antivirus Behavior-based, heuristic, and real-time antivirus protection]( https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-in-windows-10) | Always-on scanning for file and process threats which may not be detected as malware.
```

JohanFreelancer9 commented on microsoft-365/managed-desktop/intro/technologies.md:
```diff
@@ -32,12 +32,31 @@ Win32 Applications |	Teams is not shipped with the device, but is packaged and p
 
-## Windows 10 Enterprise E3 or E5
+## Windows 10 Enterprise E3, E5 and Microsoft Defender for Endpoint
+ --- | --- 
```
```suggestion
 --- | --- |
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

yogkumgit commented:
@dannoakes

yogkumgit commented:
@dannoakes Please implement the feedback so that merge can be initiated. Thanks

dannoakes commented:
@yogkumgit please confirm the feedback.

JohanFreelancer9 reviewed (changes requested):
@dannoakes - please check, if you look at the rich diff, the table does not render correctly.

dannoakes committed:
Merge branch 'public' into patch-4

yogkumgit commented:
@JohanFreelancer9

jaimeo commented:
@dannoakes your revised heading is unclear. Are there three things in the list? (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5, 3) Microsoft Defender for Endpoint.) Or two: (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5 + Microsoft Defender for Endpoint)?

yogkumgit commented:
@dannoakes Please reply to this comment. Thanks

dannoakes commented:
> @dannoakes your revised heading is unclear. Are there three things in the list? (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5, 3) Microsoft Defender for Endpoint.) Or two: (1: Windows 10 Enterprise E3, 2) Windows 10 Enterprise E5 + Microsoft Defender for Endpoint)?

It is s single heading "Windows 10 Enterprise E3, E5 and Microsoft Defender for Endpoint" for the table. It lists the 3 product SKUs a customer may have purchased. If it is simpler change to "Windows 10 Enterprise E5 or E3 with Microsoft Defender for Endpoint"

This is no different to the "Office 365 E3 or E5" heading for the above table, this is describing 2 product SKUs a customer may have purshased.

dannoakes committed:
Update technologies.md

Updated heading: Windows 10 Enterprise E5 or E3 with Microsoft Defender for Endpoint

dannoakes committed:
Update technologies.md

Correct error in table "Recommended"

jaimeo commented:
Thanks! I think that slight rewording makes it clearer, especially for people who are just scanning.
